Countries/DetailsViewController.swift

```diff
@@ -9,51 +10,182 @@ import UIKit
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    private let mainStackView: UIStackView = {
         let stackView = UIStackView()
         stackView.axis = .vertical
         stackView.distribution = .fillEqually
         stackView.translatesAutoresizingMaskIntoConstraints = false
         return stackView
     }()
     
+    let defaultView: UIView = {
+        let view = CountryTableViewHeader()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    private let headerView: DetailsVCHeader = {
+        let view = DetailsVCHeader()
+        view.contentMode = .scaleAspectFit
+        
+        if UIDevice.current.userInterfaceIdiom == .phone {
+            view.backgroundColor = Constants.backgroundColor
+        }
+        
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    let flagImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+    
     var country: CountriesApiQuery.Data.Country?
     
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
-        createLabelsAndFlag()
-        setupStackView()
+        
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            defaultImage()
+        }
+        
+        if country != nil {
+            defaultView.isHidden = true
+            createLabelsAndFlag()
+            configureFlag()
+            configureContents()
+        }
+    }
+    
+    private func defaultImage() {
+        view.addSubview(defaultView)
+        
+        NSLayoutConstraint.activate([
+            defaultView.topAnchor.constraint(equalTo: view.topAnchor, constant: Constants.topAnchorConstant),
+            defaultView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.leadingAnchorConstant),
+            defaultView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Constants.trailingAnchorConstant),
+            defaultView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -Constants.bottomAnchorConstant)
+        ])
+    }
+    
+    private func configureFlag() {
+        
+        guard let country = country else { return }
+        
+        let layer = CALayer()
+        layer.cornerRadius = Constants.cornerRadius
+        flagImageView.layer.addSublayer(layer)
+        
+        let image = UIImage(named: country.code.lowercased())
+        flagImageView.image = image
     }
     
-    private func setupStackView() {
-        view.addSubview(stackView)
+    private func configureContents() {
+        
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            constraintsForIpad()
+        } else {
+            constraintsForIphone()
+        }
```

Comment:
What is the difference between iPad and iPhone constraints?

Reply:
there is containerView for iPad similar to the design